lib/rules/__tests__/index.test.mjs

```diff
@@ -46,7 +46,8 @@ describe('fixable rules', () => {
 	});
 });
 
-describe('deprecated rules', () => {
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip('deprecated rules', () => {
```

Comment:
Now, the deprecated rules array is empty, so this ends up being an error. Should we leave this as skipped, delete the test entirely, etc>?

Reply:
I want to leave the case for future deprecated rules. How about leaving a reason as a comment like this?

```js
// eslint-disable-next-line jest/no-disabled-tests -- Leave the case for future deprecated rules.
```